testing/karma-sw.conf.js

```diff
@@ -0,0 +1,80 @@
+// Karma configuration
+// Generated on Thu Apr 16 2015 10:57:23 GMT+0100 (BST)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '../',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['sw-mocha', 'sinon', 'chai'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+      {pattern: 'lib/**/*.js', included: false}
+    ],
+
+
+    // list of files to exclude
+    exclude: [
+    ],
+
+
+    // preprocess matching files before serving them to the browser
+    // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
+    preprocessors: {
+    },
+
+
+    // test results reporter to use
+    // possible values: 'dots', 'progress'
+    // available reporters: https://npmjs.org/browse/keyword/karma-reporter
+    reporters: ['progress'],
+
+
+    // web server port
+    port: 9876,
+
+
+    // enable / disable colors in the output (reporters and logs)
+    colors: true,
+
+
+    // level of logging
+    // possible values: config.LOG_DISABLE || config.LOG_ERROR || config.LOG_WARN || config.LOG_INFO || config.LOG_DEBUG
+    logLevel: config.LOG_INFO,
+
+
+    // enable / disable watching file and executing tests whenever any file changes
+    autoWatch: true,
+
+
+    // start these browsers
+    // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
+    browsers: ['Chrome', 'NightlySW'],
+
+    customLaunchers: {
+      'NightlySW': {
+        base: 'Firefox',
```

Comment:
Same here. Indeed, with the `FIREFOX_BIN` fix and the [PR for allowing adding flags](https://github.com/karma-runner/karma-firefox-launcher/pull/36) is enough.